AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -33,6 +33,7 @@ reddit.com#$#body.scroll-disabled { overflow: visible!important; position: stati
 reddit.com##body > div[id="container"] > div[class="App"] > div[class="AppMainPage"] > div[class="NavFrame"] > div[class="XPromoPopup active"]
 !
 scribd.com##.auto__doc_page_webpack_view_in_store_promo
+kreiszeitung.de#?#.id-SiteMain > div.id-g > div[class] > div.id-Container:has(> div.id-Container-content > div.id-LB > div[class^="id-LB"] > div[class^="id-Teaser"] > a[href^="https://itunes.apple.com/de/app/mediengruppe-kreiszeitung/"])
```

Comment:
Not shown in code. iOS does not support ext-CSS, so making it the search parameter is a bit contraproductive.

Reply:
for example <a href="https://www.kreiszeitung.de/lokales/diepholz/bassum-ort51127/corona-physiotherapeuten-haben-weniger-patienten-arbeiten-aber-weiter-13612348.html">here<a>
<details>
<summary>Screenshot</summary>
<img src="https://user-images.githubusercontent.com/18536328/77901114-bc170c80-7287-11ea-9ba7-a49e856fbaf5.png">
</details>

Reply:
> Not shown in code. iOS does not support ext-CSS, so making it the search parameter is a bit contraproductive.

It looks like that this element can not be removed safely at iOS in any case and the element looks the same on all platforms.

Reply:
OK.